packages/astro/src/runtime/client/hmr.ts

```diff
@@ -29,6 +48,18 @@ if (import.meta.hot) {
 					link.replaceWith(link.cloneNode(true));
 				}
 			}
+			if (file.acceptedPath.includes('astro&type=style')) {
```

Comment:
For .astro styles, they are now injected as inline `<style>` blocks (similar to what Vite does internally, including with Svelte styles)